app/rbkitConnection.js

```diff
@@ -0,0 +1,48 @@
+import ZMQ from 'zmq';
+import MsgPack from 'msgpack';
+
+const RbkitConnection = (() => {
+  let ip;
+  let requester;
+  let subscriber;
+  let rawProfilingData = [];
```

Comment:
This file is some sort of a helper; so doesn't fit in `redux` and `components`. Shouldn't just be under `app` though.